services/src/pro/contexts/migrations/migration_0010_s2_stack_time_buffers.rs

```diff
@@ -0,0 +1,32 @@
+use async_trait::async_trait;
+use tokio_postgres::Transaction;
+
+use super::database_migration::{ProMigration, ProMigrationImpl};
+use crate::{contexts::Migration0010S2StacTimeBuffers, error::Result};
+
+#[async_trait]
+impl ProMigration for ProMigrationImpl<Migration0010S2StacTimeBuffers> {
+    async fn pro_migrate(&self, tx: &Transaction<'_>) -> Result<()> {
+        // add priority column to layer_providers table and move description column to the definition types
+        tx.batch_execute(
+            r#"
+            CREATE TYPE "StacQueryBuffer" AS (start_seconds bigint, end_seconds bigint);
+
+            ALTER TYPE "SentinelS2L2ACogsProviderDefinition"
+            ADD ATTRIBUTE query_buffer "StacQueryBuffer";            
+        "#,
+        )
+        .await?;
+
+        tx.batch_execute(
+            r"
+            UPDATE pro_layer_providers
+            SET
+                definition.sentinel_s2_l2_a_cogs_provider_definition.query_buffer.start_seconds = -60,
```

Comment:
does it make sense that this is negative? I think it should be positive...

Reply:
changed it to positive numbers and subtraction (sorry forgot extra commit)